### PR TITLE
[LCI] Implement Ojer Axonil, Deepest Might

### PR DESCRIPTION
**Pull request by Susucre**
Based on top of #11193 for the set file to be created.

### CODE REVIEW

**Comment by Susucre** [comment]
[[Ojer Axonil]]

**Comment by Susucre** [comment]
Not sure I did anything weird, but transforming back into the creature from the returned land is not in the right row:
image

Functionality is good from there (a creature not a land, can attack, etc...)

**Review by xenohedron — COMMENTED** [review]
the new card logic looks good to me. can you rebase and clean up the PR?

File: Mage.Sets/src/mage/cards/t/TempleOfPower.java [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    @Override
+    public boolean apply(Game game, Ability source) {
+        TempleOfPowerWatcher watcher = game.getState().getWatcher(TempleOfPowerWatcher.class);
+        return watcher != null
+                && 4 <= watcher.damageForPlayer(source.getControllerId());
+    }
+
+    @Override
+    public String toString() {
+        return "if red sources you controlled dealt 4 or more noncombat damage this turn and only as a sorcery";
```
[the remark this answers is not in the document]
Comment (Susucre) [reply Mage.Sets/src/mage/cards/t/TempleOfPower.java:84]: You are sure you want to modify the effect's text generation for that?
There are only 4 cards using that wording https://scryfall.com/search?q=o%3A%22and+only+as+a+sorcery%22
Detention Vortex does not use the same effect, Speaker of the Heavens could be refactored, but currently use a custom trigger, and Heron-Blessed Geist is adding that text in its Condition.

**Review by xenohedron — APPROVED** [review]

**Pull request merged**